edit/networking.k8s.io.networkpolicy/PolicyRuleTarget.vue

```diff
@@ -237,6 +235,13 @@ export default {
           />
         </div>
       </div>
+      <div v-if="invalidCidrs.length" class="row mb-10">
+        <div class="col span-12">
+          <Banner color="error">
+            <t v-if="invalidCidrs.length" k="networkpolicy.rules.ipBlock.invalidExceptionCidrs" />{{ invalidCidrs.join(', ') }}
```

Comment:
Nit: the `v-if` condition is already checked in the `div` above

Reply:
Good catch! I've removed the check here and on the text for `invalidCidr` as well